regression/daily/runPteTestSuite.sh

```diff
@@ -34,5 +22,13 @@ if [ ! -d "node_modules" ];then
     fi
 fi
 
-cd $DAILYDIR && GO111MODULE=on  ginkgo -v && echo "------> PTE tests completed"
-archivePTE
+cd $DAILYDIR && GO111MODULE=on ginkgo -v
+StatusPteNL=$?
+
+if [ $StatusPteNL == 0 ]; then
+    echo "------> PTE/NL tests completed"
+else
+    echo "------> PTE/NL tests failed with above errors"
+    exit 1
+fi
+cp $FabricTestDir/tools/PTE/CITest/Logs/*.log $DAILYDIR
```

Comment:
Given that we aren't publishing the logs anymore, why are we bothering to move them at all?